src/internal/connector/graph_connector.go

```diff
@@ -220,6 +220,8 @@ func (gc *GraphConnector) setTenantSites(ctx context.Context) error {
 
 var errKnownSkippableCase = errors.New("case is known and skippable")
 
+const personalSitePath = "sharepoint.com/personal/"
```

Comment:
this is a bit more of a long-term question, but do we have any way to know if this is a locale independent way to specify personal sites?

Reply:
Good question. Unfortunately not; the [site](https://learn.microsoft.com/en-us/graph/api/resources/site?view=graph-rest-1.0) sttruct is pretty sparse.  The next closest data point we have is `siteCollection.hostname`, where personal sites all have the domain `<tenant>-my.sharepoint.com`.  But, then again, that could be subject to locale as well.